eclipse/ide-target-platform/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>com.gwtplugins.eclipse</groupId>
     <artifactId>trunk</artifactId>
-    <version>4.1.0-SNAPSHOT</version>
+    <version>4.2.0-SNAPSHOT</version>
```

Comment:
If I get it right, this "4.2.0-SNAPSHOT" is derived from parent POM if not specified at all.
Disclaimer: not a maven expert